netjsonconfig/backends/airos/converters.py

```diff
@@ -0,0 +1,809 @@
+
+
+class Iptables(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = [
+               {
+                    'sys': {
+                        'portfw':  {
+                            'status': 'disabled',
+                        },
+                        'status':  'enabled',
+                    },
+                    'status':  'disabled',
+                },
+        ]
+
+        return (('iptables', result),)
+
+
+class Netconf(AirOSConverter):
+    netjson_key = 'interfaces'
+
+    def type_to_role(self, typestr):
+        roles = {
+            'ethernet': 'mlan',
+            'bridge': 'mlan',
+        }
+        return roles.get(typestr, '')
+
+    def to_intermediate(self):
+        result = []
+        interfaces = []
+        original = get_copy(self.netjson, self.netjson_key, [])
+
+        for interface in original:
+            base = {
+                'devname':  interface['name'],
+                'status': 'enabled',  # can't disable interfaces
+                'up':  status(interface),
+                'mtu': interface.get('mtu', 1500),
+            }
+
+            # handle interface type quirks
+            if interface['type'] == 'ethernet' and '.' not in interface['name']:
+                base['autoneg'] = 'enabled'
+                base['flowcontrol'] = {
+                        'rx': {
+                            'status': 'enabled',
+                        },
+                        'tx': {
+                            'status': 'enabled',
+                        },
+                    }
+
+            if interface['type'] == 'wireless':
+                base['devname'] = interface['wireless']['radio']
+
+            addresses = interface.get('addresses')
+
+            if addresses:
+                # for every address policy put a
+                # configuration
+                for addr in addresses:
+                    temp = deepcopy(base)
+
+                    if addr.get('management'):
+                        temp['role'] = self.type_to_role(interface['type'])
+
+                    # handle explicit address policy
+                    if addr['proto'] == 'dhcp':
+                        temp['autoip'] = {}
+                        temp['autoip']['status'] = 'enabled'
+                    else:
+                        ip_and_mask = '%s/%d' % (addr['address'], addr['mask'])
+                        network = ip_interface(ip_and_mask)
+                        temp['ip'] = str(network.ip)
+                        temp['netmask'] = str(network.netmask)
+
+                    interfaces.append(temp)
+            else:
+                # an interface without address
+                # is still valid with these defaults values
+                base['autoip'] = {
+                        'status': 'disabled',
+                    }
+                interfaces.append(base)
+
+        result.append(interfaces)
+        result.append({
```

Comment:
you can shorten this (and cases similar to this) to:

`result.append({'status':  'enabled'})`